src/index.html

```diff
@@ -2,16 +2,148 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label>
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label>
+          Name:
+          <input
+            type="text"
+            name="name"
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label>
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label>
+          Full date of birth:
+          <input
+            type="date"
+            name="date-of-birth"
+            required
+          >
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label>
+          E-mail:
+          <input
+            type="email"
+            name="mail"
+            required
+            minlength="5"
+            maxlength="40"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label>
+          Password: <input type="password"
+                          name="password"
+                          required
+                    >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you! </legend>
+        Do you love cats?
+        <label class="diff">
+          <input type="radio"
+                 name="catlover"
+                 value="yes"
+          >
```

Comment:
Fix everywhere, please (checklist p.3)
```suggestion
          <input 
            type="radio"
            name="catlover"
            value="yes"
          >
```